src/components/headerMenu.js

```diff
@@ -2,9 +2,43 @@ import { Link } from "gatsby";
 import React from "react";
 
 export const HeaderMenu = () => {
+  let isOpen = false;
+
+  const menuToggleFunc = () => {
+    const container = document.querySelector("#headerMenuContainer");
+    if (isOpen) {
+      container.style.display = "none";
+      isOpen = false;
+    } else {
+      container.style.display = "block";
+      isOpen = true;
+    }
+  };
+
+  const onKeyMenuToggle = (keydownEvent) => {
+    // 13 - Enter key code
```

Comment:
Declare the magic number as constant with a descriptive name.

Reply:
Done